skfda/representation/basis.py

```diff
@@ -331,7 +335,9 @@ def __repr__(self):
 
     def __eq__(self, other):
         """Equality of Basis"""
-        return type(self) == type(other) and _same_domain(self.domain_range, other.domain_range) and self.nbasis == other.nbasis
+        return (type(self) == type(other)
```

Comment:
This will have to be overridden when you implement the custom basis for FPCA, right?

Reply:
Yes